scripts/dream.py

```diff
@@ -14,12 +14,30 @@
 from ldm.dream.server import DreamServer, ThreadingDreamServer
 from ldm.dream.image_util import make_grid
 from omegaconf import OmegaConf
+import torch
```

Comment:
The torch library takes a significant time to load; it's particularly a problem in my environment where the libraries are being loaded from a shared network mount. I don't like having to load torch just to print out the `dream.py -h` help message, and for this reason torch and other big libraries are only imported later on in generate.py.